tests/main/snap-info/task.yaml

```diff
@@ -4,9 +4,6 @@ summary: Check that snap info works
 systems: [-ubuntu-core-18-*]
 
 prepare: |
-    . $TESTSLIB/pkgdb.sh
-    distro_install_package python3-yaml
```

Comment:
You still need to add that to pkg_dependencies on ubuntu. Other distros seem to have it there already.